docs/core/preview3/tools/csproj.md

```diff
@@ -19,19 +19,80 @@ This document outlines the changes that were added to the csproj files as part o
+
+The table below shows which element and which globs are both included and excluded in the SDK: 
+
+| Element          	| Include glob                           	| Exclude glob                                     	            | Remove glob             	 |
+|-------------------|-------------------------------------------|---------------------------------------------------------------|----------------------------|
+| Compile          	| \*\*/\*.cs (or other language extensions) | \*\*/\*.user;  \*\*/\*.\*proj;  \*\*/\*.sln;  \*\*/\*.vssscc 	| N/A                     	 |
+| EmbeddedResource 	| \*\*/\*.resx                             	| \*\*/\*.user; \*\*/\*.\*proj; \*\*/\*.sln; \*\*/\*.vssscc     | N/A                     	 |
+| None             	| \*\*/\*                                  	| \*\*/\*.user; \*\*/\*.\*proj; \*\*/\*.sln; \*\*/\*.vssscc     | - \*\*/\*.cs; \*\*/\*.resx |
+
+If you have globs in your project and you try to build it using the newest SDK, you'll get the following error:
+
+> Duplicate Compile items were included. The .NET SDK includes Compile items from your project directory by default. You can either remove these items from your project file, or set the 'EnableDefaultCompileItems' property to 'false' if you want to explicitly include them in your project file. 
+
+In order to get around this error, you can either remove the explicit Compile items that match the ones listed above, or you can set the `<EnableDefaultCompileItems>` property to false, like this:
+
+```xml
+<PropertyGroup>
+    <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
+</PropertyGroup>
+```
+Setting this property to `false` will override implicit inclusion and the behavior will revert back to the previous SDKs where you had to specify the default globs in your project. 
+
+This change does not modify the main mechanics of other includes. However, if you wish to specify, for example, some files to get published with your app, you can still use the known mechanisms in `csproj` for that (for example, the `<Content>` element).
+
+### Recommendation
+Moving forward, our recommendation is for you to remove the above default globs from your project and only add globs file paths for those artifacts that your app/library needs for various scenarios (runtime, NuGet packaging, etc.)
+
+
 ## Additions
 
-* PackageReference
+### Sdk property 
+As part of this work, we've added a new property to the `<Project>` element of the `csproj` file that is called SDK. The SDK property lists out what SDK will be used in this project. The SDK, as the [layering document](layering.md) describes, is a set of MSBuild [tasks]() and [targets]() that can build .NET Core code. We ship two main SDKs with the .NET Core tools:
```

Comment:
- shouldn't the name of the property be Sdk not SDK? Also inline code those when referring to the property name.
- missing links here.
- .NET Core code -> .NET Core projects
- We ship two main SDKs with the .NET Core tools: -> There are two main SDKs with the .NET Core tools: